### PR TITLE
Add sponsors to the main conf page

### PR DESCRIPTION
**Pull request by plaindocs**
Replaces #126

### CODE REVIEW

**Comment by plaindocs** [comment]
I'm not 100% sure what I'm doing with the CSS grid classes, and the images don't display locally which makes it hard to figure out. Can I set branches to display on https://readthedocs.org/projects/writethedocs-www/builds/ ?

I need to take the rst headings out of the include.

File: docs/conf/na/2017/sponsor.rst [hunk 1]
```diff
@@ -7,39 +7,4 @@ We are seeking corporate partners to help us create the best conference
...
-
-    <div class="row row-sponsors">
-      <!-- Patron -->
-      <div class="col-md-8 col-md-offset-2 col-sm-8 col-sm-offset-2">
-        <a href="https://readthedocs.com">
-        <img src="/_static/img/2016/sponsors/read-the-docs.png" width=400/></a>
-      </div>
-   </div>
\ No newline at end of file
+.. include:: ../../../include/conf/2017-na-sponsors.rst
```
Comment (ericholscher) [inline docs/conf/na/2017/sponsor.rst]: You can use absolute includes, don't need to be relative.

File: docs/conf/na/2017/index.rst [hunk 2]
```diff
@@ -122,10 +122,32 @@
...
+                      <span class="subheader__yellow"></span>
+                      <h2 class="subheader__header">Sponsors</h2>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="row">
+                  <div class="col-xs-12">
+
+.. include::   ../../../include/conf/2017-na-sponsors.rst
```
Comment (ericholscher) [inline docs/conf/na/2017/index.rst]: You can use absolute includes, don't need to be relative.

File: docs/_templates/2017/na-menu.html [hunk 3]
```diff
@@ -58,7 +58,6 @@
       </a>
     </div>
   </div>
-  </div>
```
Comment (ericholscher) [inline docs/_templates/2017/na-menu.html:61]: Was this just broken?

**Review by ericholscher — APPROVED** [review]
Looks good overall, just a few nits.

File: docs/_templates/2017/na-menu.html (hunk 3 above)
Comment (plaindocs) [reply docs/_templates/2017/na-menu.html:61]: I think so, yes.

**Comment by plaindocs** [comment]
Damn, was just coming to suggest not merging this. :-p. It does not seem to be live anyway?

**Comment by ericholscher** [comment]
Yea, it exploded when I merged it and had to kill it.

**Comment by plaindocs** [comment]
Ah, I missed the revert. Stay tuned for new and improved PR number 3.

**Pull request merged**